tests/test_search.py

```diff
@@ -1,7 +1,54 @@
+from sharepa.search import ShareSearch
 from sharepa.search import basic_search
 
+import elasticsearch_dsl
+import types
+
 
 def test_basic_search():
     results = basic_search.execute()
     assert results.hits
     assert results.aggregations
+
+
+def test_no_title_search():
+    my_search = ShareSearch()
+
+    my_search = my_search.query(
+        'query_string',
+        query='NOT title:*',
+        analyze_wildcard=True
+    )
+    results = my_search.execute()
+    for result in results:
+        assert not result.get('title')
+
+
+def test_execute():
+    my_search = ShareSearch()
+
+    result = my_search.execute()
+    assert isinstance(result, elasticsearch_dsl.result.Response)
```

Comment:
Better to test the format and content of the return values, rather than the return type (since a Response is returned even on an error). Can use ```vcr.py``` to ensure that the results don't change over time.